MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillImpl.kt

```diff
@@ -113,22 +137,22 @@ internal class AddressAutofillImpl(
 
             return AddressAutofillImpl(
                 accessToken = accessToken,
-                searchEngine = engine,
+                autofillEngine = engine,
                 activityReporter = getUserActivityReporter(accessToken)
             )
         }
 
-        private fun List<BaseSearchResult>.toAddressAutofillSuggestions() = mapNotNull { it.toAddressAutofillSuggestion() }
+        private fun List<BaseSearchSuggestion>.toAddressAutofillSuggestionList()  = mapNotNull { it.toAddressAutofillSuggestion() }
```

Comment:
Given that we have `AddressAutofillResultFactory` now, it makes sense to move these functions there.